infrastructure/lambda/template.yaml

```
@@ -757,6 +757,34 @@ Resources:
       FunctionName: !GetAtt IssueCredentialFunction.Arn
       Principal: apigateway.amazonaws.com
 
+  KBVQuestionFunctionPermissionAlias:
```

```suggestion
  KBVQuestionFunctionAliasPermission:
```
This is a permission to invoke the function alias, which the name should reflect.
This also applies to the other alias permissions added in this PR.